### DISCUSSION TITLE
Multiple Sources vs Idle Blackout

### DISCUSSION THREAD

**Discussion by Jackalolz** [JPersson77/LGTVCompanion#225] [General]
Question to the creator & community.

My workstation setup is primarily a Windows Desktop and on occasion alternating between my desktop and a Macbook Laptop. 

With Windows being my primary source, I run Idle Blackout mode to kick in 15 minutes because I'll often walk away from my computer and the blackout saves me a great deal of stress of burn in.

 The issue I have is when running both operating systems through the same screen, my Windows will reach the 15-minute timer and blackout while my C3 LG television is displaying my MacBook on screen. 
 
 Is there a way for LG Companion to detect what source is active from WebOS for the Idle counter to be paused or disabled?
 Any solution other than booting up the program and disabling the timer when I'm using my Mac? Will there be a MacOS to override the 
 idle counter?

**Comment by JPersson77** [comment]
Hi @Jackalolz 

I assume you have the PC and Mac connected to different HDMI-inputs. Then you can enable this option in the device configuration, which should solve your issue:

![image](https://github.com/JPersson77/LGTVCompanion/assets/77928210/dbb1081d-57d2-42fe-94c7-bd64e0888086)

If the PC or Mac is connected through a docking station then you may want to consider enabling the monitor topology support feature in the global options. This will let the app be aware of attached/detached monitors.

![image](https://github.com/JPersson77/LGTVCompanion/assets/77928210/6a1d754f-7100-48f5-a90f-f6d70cc7b3f1)

Let me know if the above helps. You can also control other aspects of the app via command line but I think this is what you asked for

**Reply by Jackalolz** [reply]
Awesome. Thanks for the support

**Comment by JPersson77** [comment]
cheers :)
